### ISSUE TITLE
memory-cache-ttl dependency

### ISSUE THREAD

**Issue by harwoodleon** [datproject/dat#823]
When running a fresh install I get 

```
  /usr/local/bin/dat -> /usr/local/lib/node_modules/dat/bin/cli.js
  
  > utp-native@1.5.2 install /usr/local/lib/node_modules/dat/node_modules/utp-native
  > node-gyp-build


> sodium-native@1.10.2 install /usr/local/lib/node_modules/dat/node_modules/sodium-native
> node-gyp-build "node preinstall.js" "node postinstall.js"


> memory-cache-ttl@1.0.51 postinstall /usr/local/lib/node_modules/dat/node_modules/memory-cache-ttl
> npm run build


> memory-cache-ttl@1.0.51 build /usr/local/lib/node_modules/dat/node_modules/memory-cache-ttl
> BABEL_ENV=production babel src --out-dir dist

sh: babel: command not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! memory-cache-ttl@1.0.51 build: `BABEL_ENV=production babel src --out-dir dist`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the memory-cache-ttl@1.0.51 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

┌──────────────────────────────────────────────────────────┐
│                 npm update check failed                  │
│           Try running with sudo or get access            │
│           to the local update config store via           │
│ sudo chown -R $USER:$(id -gn $USER) /Users/xxxx/.config │
└──────────────────────────────────────────────────────────┘
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! memory-cache-ttl@1.0.51 postinstall: `npm run build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the memory-cache-ttl@1.0.51 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

There was a recent update on memory-cache-ttl

https://github.com/tiborv/memory-cache-ttl/commit/3582b8ce91c005f52b489f4436acf0038d4951c7

I wonder if this has broken the install?

**Comment by mafintosh** [comment]
Note this is fixed by datprotocol/dat-dns#3 being merged + released

**Comment by mafintosh** [comment]
fixed in dat-dns
